docs/PROPSMETHODS.md

```diff
@@ -98,6 +98,16 @@ If you have any inputs inside your cont and you want to manage how to dismiss th
 | ---------------------------------- | -------- | -------- |
 | enum('never', 'always', 'handled') | No       | `never`  |
 
+### `keyboardAvoidingBehavior`
+
+Define the behavior of the modal when keyboard is active.
+
+If you have any inputs inside your cont and you want to manage how the view should change when the keyboard is active. See [`react-native` documentation](https://facebook.github.io/react-native/docs/keyboardavoidingview#behavior) for more informations.
+
+| Type                                  | Required | Default  |
```

Comment:
You can remove the default column here, since there isn't

Reply:
Done